consoles/sshVirtsh.pm

```diff
@@ -325,111 +325,169 @@ sub add_interface {
     return;
 }
 
+sub to_byte {
+    my $sp              = shift;
+    my $size_multiplier = {
+        b => 1,
+        k => 1024, K => 1024,
+        M => 1024 * 1024,
+        G => 1024 * 1024 * 1024,
+        T => 1024 * 1024 * 1024 * 1024,
```

Comment:
In Perl there's a operator for computing an exponentiation, e.g.:
```suggestion
        T => 1024 ** 3,
```
But `perl-Number-Format` is already in Factory so you might just depend on that.